diffus-derive/src/lib.rs

```diff
@@ -235,13 +247,13 @@ pub fn derive_diffus(input: proc_macro::TokenStream) -> proc_macro::TokenStream
                 }
             });
 
-            proc_macro::TokenStream::from(quote! {
-                #vis enum #edited_ident<'a> {
+            let result = quote! {
+                #vis enum #edited_ident#lifetime {
```

Comment:
Supernit: Perhaps space before `#` here and on line `256`

Reply:
I thought about it when I wrote it, but I think you are right, much more important that `#lifetime` is readable